app/assets/javascripts/discourse/app/components/d-multi-select.gjs

```diff
@@ -77,7 +77,7 @@ export default class DMultiSelect extends Component {
 
   @action
   focus(input) {
-    input.focus();
+    input.focus({ preventScroll: true });
```

Comment:
not directly relevant to this PR, but opening a multi select was causing a page scroll